atlas/commands/bootstrap/templates/cmd/server/config.go.gotmpl

```diff
@@ -0,0 +1,124 @@
+package main
+
+import "github.com/spf13/pflag"
+
+const (
+	// configuration defaults support local development (i.e. "go run ...")
+
+	// Server
+	defaultServerAddress = "0.0.0.0"
+	defaultServerPort    = "9090"
+
+	// Gateway 
+	defaultGatewayEnable      = {{ if .WithGateway }}true{{ else }}false{{ end }}
+	defaultGatewayAddress     = "0.0.0.0"
+	defaultGatewayPort        = "8080"
+	defaultGatewayURL         = "/{{ .Name | URL }}/v1/"
+	defaultGatewaySwaggerFile = "pkg/pb/service.swagger.json"
+	defaultGatewaySwaggerUI   = "./apidoc-poc/swagger-ui-dist/"
```

Comment:
This should not be in template. `swagger-ui-dist` is related to swagger ui poc. Normal applications will not expose swagger UI, only their swagger.json.